### PR TITLE
portfolio part 2

### PR DESCRIPTION
**Pull request by radhikaprabu**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+							<div class="icon">
+								<i class="fa fa-phone-square fa-3x fa-fw icon-item"></i>
+								<span class="desktop hidden-xs"> 763 219 0566 </span>
+								<span class="visible-xs-inline"><a href="#" class="mobile"> 763 219 0566 </a></span>
+							</div>
+						</li>
+						<li>
+							<div class="icon">
+								<i class="fa fa-envelope-square fa-3x fa-fw icon-item"></i>
+								<a href="rrao001@my.hennepintech.edu" class="email"> E-MAIL </a>
```
Comment (mbMosman) [inline index.html]: This should be a mailto link.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+		<div class="container">
+			<h1><span class="label label-default">CONTACT</span></h1>
+			<div class="row">
+				<div class="col-xs-12 col-sm-6 col-md-6">
+					<ul class="list" style="list-style-type:none">
+						<li>
+							<div class="icon">
+								<i class="fa fa-phone-square fa-3x fa-fw icon-item"></i>
+								<span class="desktop hidden-xs"> 763 219 0566 </span>
+								<span class="visible-xs-inline"><a href="#" class="mobile"> 763 219 0566 </a></span>
```
Comment (mbMosman) [inline index.html]: This should be a tel link, # will just go to the top of the page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+			<div class="collapse navbar-collapse" id="mynavbar">
+				<ul class="nav navbar-nav">
+					<li>
+						<a href="#about">About</a>
+					</li>
+					<li>
+						<a href="#skills">Skills</a>
+					</li>
+					<li>
+						<div class="visible-md-inline visible-lg-inline" id="brand">
```
Comment (mbMosman) [inline index.html]: I did remember this and spent a little time with it, but unfortunately I can't find a way to do what you want with it either.  :frowning:

**Comment by mbMosman** [comment]
Really great job overall.

File: index.html (hunk 3 above)
Comment (radhikaprabu) [inline index.html]: Thanks Mary!

**Pull request closed without merging**